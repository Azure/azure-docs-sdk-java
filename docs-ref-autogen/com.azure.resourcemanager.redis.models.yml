### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.redis.models"
fullName: "com.azure.resourcemanager.redis.models"
name: "com.azure.resourcemanager.redis.models"
summary: "Package containing the data models for RedisManagementClient."
classes:
- "com.azure.resourcemanager.redis.models.CheckNameAvailabilityParameters"
- "com.azure.resourcemanager.redis.models.DefaultName"
- "com.azure.resourcemanager.redis.models.ErrorAdditionalInfo"
- "com.azure.resourcemanager.redis.models.ErrorDetail"
- "com.azure.resourcemanager.redis.models.ExportRdbParameters"
- "com.azure.resourcemanager.redis.models.ImportRdbParameters"
- "com.azure.resourcemanager.redis.models.ManagedServiceIdentity"
- "com.azure.resourcemanager.redis.models.ManagedServiceIdentityType"
- "com.azure.resourcemanager.redis.models.NotificationListResponse"
- "com.azure.resourcemanager.redis.models.OperationDisplay"
- "com.azure.resourcemanager.redis.models.OperationListResult"
- "com.azure.resourcemanager.redis.models.OperationStatusResult"
- "com.azure.resourcemanager.redis.models.PrivateEndpoint"
- "com.azure.resourcemanager.redis.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.redis.models.PrivateEndpointConnectionProvisioningState"
- "com.azure.resourcemanager.redis.models.PrivateEndpointServiceConnectionStatus"
- "com.azure.resourcemanager.redis.models.PrivateLinkResourceListResult"
- "com.azure.resourcemanager.redis.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.redis.models.ProvisioningState"
- "com.azure.resourcemanager.redis.models.PublicNetworkAccess"
- "com.azure.resourcemanager.redis.models.RebootType"
- "com.azure.resourcemanager.redis.models.RedisCommonProperties"
- "com.azure.resourcemanager.redis.models.RedisConfiguration"
- "com.azure.resourcemanager.redis.models.RedisCreateParameters"
- "com.azure.resourcemanager.redis.models.RedisFirewallRuleCreateParameters"
- "com.azure.resourcemanager.redis.models.RedisFirewallRuleListResult"
- "com.azure.resourcemanager.redis.models.RedisInstanceDetails"
- "com.azure.resourcemanager.redis.models.RedisLinkedServer"
- "com.azure.resourcemanager.redis.models.RedisLinkedServerCreateParameters"
- "com.azure.resourcemanager.redis.models.RedisLinkedServerWithPropertiesList"
- "com.azure.resourcemanager.redis.models.RedisListResult"
- "com.azure.resourcemanager.redis.models.RedisPatchScheduleListResult"
- "com.azure.resourcemanager.redis.models.RedisRebootParameters"
- "com.azure.resourcemanager.redis.models.RedisRegenerateKeyParameters"
- "com.azure.resourcemanager.redis.models.RedisUpdateParameters"
- "com.azure.resourcemanager.redis.models.ScheduleEntry"
- "com.azure.resourcemanager.redis.models.Sku"
- "com.azure.resourcemanager.redis.models.SkuFamily"
- "com.azure.resourcemanager.redis.models.SkuName"
- "com.azure.resourcemanager.redis.models.TlsVersion"
- "com.azure.resourcemanager.redis.models.UserAssignedIdentity"
enums:
- "com.azure.resourcemanager.redis.models.DayOfWeek"
- "com.azure.resourcemanager.redis.models.RedisCache.RedisVersion"
- "com.azure.resourcemanager.redis.models.RedisKeyType"
- "com.azure.resourcemanager.redis.models.ReplicationRole"
interfaces:
- "com.azure.resourcemanager.redis.models.RedisAccessKeys"
- "com.azure.resourcemanager.redis.models.RedisCache"
- "com.azure.resourcemanager.redis.models.RedisCache.Definition"
- "com.azure.resourcemanager.redis.models.RedisCache.DefinitionStages"
- "com.azure.resourcemanager.redis.models.RedisCache.DefinitionStages.Blank"
- "com.azure.resourcemanager.redis.models.RedisCache.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.redis.models.RedisCache.DefinitionStages.WithGroup"
- "com.azure.resourcemanager.redis.models.RedisCache.DefinitionStages.WithPremiumSkuCreate"
- "com.azure.resourcemanager.redis.models.RedisCache.DefinitionStages.WithSku"
- "com.azure.resourcemanager.redis.models.RedisCache.Update"
- "com.azure.resourcemanager.redis.models.RedisCache.UpdateStages"
- "com.azure.resourcemanager.redis.models.RedisCache.UpdateStages.WithNonSslPort"
- "com.azure.resourcemanager.redis.models.RedisCache.UpdateStages.WithRedisConfiguration"
- "com.azure.resourcemanager.redis.models.RedisCache.UpdateStages.WithSku"
- "com.azure.resourcemanager.redis.models.RedisCachePremium"
- "com.azure.resourcemanager.redis.models.RedisCaches"
- "com.azure.resourcemanager.redis.models.RedisFirewallRule"
- "com.azure.resourcemanager.redis.models.RedisPatchSchedule"
desc: "Package containing the data models for RedisManagementClient. REST API for Azure Redis Cache Service."
metadata: {}
package: "com.azure.resourcemanager.redis.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-redis:2.19.0
