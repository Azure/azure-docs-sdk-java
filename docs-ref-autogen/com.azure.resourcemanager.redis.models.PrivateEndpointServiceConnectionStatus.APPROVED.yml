### YamlMime:JavaMember
uid: "com.azure.resourcemanager.redis.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.redis.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.redis.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.redis.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.redis.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-redis:2.4.0
